config/deploy.js

```diff
@@ -6,6 +6,7 @@ module.exports = function (deployTarget) {
     build: {},
 
     git: {
+      enabled: true,
```

Comment:
Restored the option to match the "before" code:

https://github.com/ember-intl/ember-intl/blob/4a61ce6d7f64a29f15d5aced28015c16761c28fe/config/deploy.js#L8-L9